sql/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java

```diff
@@ -1047,4 +1040,95 @@ public void testGroupByOnClusteredByColumnPartitioned() throws Exception {
                 "1| 44\n" +
                 "1| 45\n"));
     }
+
+    @Test
+    public void groupBySortOnGroupingKey() throws Exception {
```

Comment:
do we really need so many new tests? There should've been some already that sort on group by key